test/functional/config/example_dut_config.yml

```diff
@@ -14,6 +14,15 @@ type: "local"
 allow_disk_autoselect: False
 working_dir: "/tmp/open-cas-linux/"
 
+# CAS version to test.
+# This version will be installed on DUT before test. Can be any string
+# digestible by git-checkout. If not specified, current state of local
+# repo (HEAD) will be used as version to install.
+# NOTE: Make sure to have a local master branch synced with upstream
+# as the local repo will be taken for version reference. To do this
+# simply run: `git fetch origin master:master`
+cas_version: "master"
```

Comment:
What if I don't need the newest OCL version on my DUT and I want to avoid reinstallation? Is is possible to skip it without putting the current commit version into dut config file? If not, I think there should be such an option available.

Reply:
I assumed that you always want to run a test on a particular version of CAS, will it be the newest master, a version tag or just a version that you're checked out on your local repo from which you run that test. If this version is already installed, there is no reinstallation.
This patch eliminates the risk of using a random CAS version installed on DUT in the past.

Is there a scenario in which using such random version would be beneficial?

Reply:
Or maybe your question is about using your "current commit version" of CAS? If so, you just need to remove or comment out the 'cas_version' in your dut_config and then your current commit from your local repo will be taken as version to check (and install if not already).